src/diagnostics/core_diagnostics.jl

```diff
@@ -733,3 +733,315 @@ add_diagnostic_variable!(
     comments = "Elevation of the horizontal coordinates",
     compute! = compute_orog!,
 )
+
+###
+# cloud liquid water specific humidity (1d)
+###
+compute_clwvi!(out, state, cache, time) =
+    compute_clwvi!(out, state, cache, time, cache.atmos.moisture_model)
+compute_clwvi!(_, _, _, _, model::T) where {T} =
+    error_diagnostic_variable("clwvi", model)
+
+function compute_clwvi!(
+    out,
+    state,
+    cache,
+    time,
+    moisture_model::T,
+) where {T <: Union{EquilMoistModel, NonEquilMoistModel}}
+    thermo_params = CAP.thermodynamics_params(cache.params)
+    if isnothing(out)
+        out = zeros(axes(Fields.level(state.f, half)))
+        Operators.column_integral_definite!(
+            out,
+            state.c.ρ .* cache.precomputed.cloud_diagnostics_tuple.q_liq,
+        )
+        return out
+    else
+        Operators.column_integral_definite!(
+            out,
+            state.c.ρ .* cache.precomputed.cloud_diagnostics_tuple.q_liq,
+        )
+    end
+end
+
+add_diagnostic_variable!(
+    short_name = "clwvi",
+    long_name = "Condensed Water Path",
+    standard_name = "atmosphere_mass_content_of_cloud_condensed_water",
+    units = "kg m-2",
+    comments = """
+    Mass of condensed (liquid + ice) water in the column divided by the area of the column 
+    (not just the area of the cloudy portion of the column). Includes precipitating hydrometeors
+    ONLY if the precipitating hydrometeors affect the calculation of radiative transfer in model.
```

Comment:
```suggestion
    Mass of condensed (liquid + ice) water in the column divided by the area of the column 
    (not just the area of the cloudy portion of the column). It doesn't include precipitating hydrometeors.
```
(In general, for the comments, let's define what is actually calculated instead of just copying it from the cmip table)